spec/lib/evm_database_ops_spec.rb

```diff
@@ -6,7 +6,7 @@
       @connect_opts = {:username => 'blah', :password => 'blahblah', :uri => "smb://myserver.com/share"}
       @db_opts =      {:dbname => 'vmdb_production', :username => 'root'}
       MiqSmbSession.stub(:runcmd)
-      MiqSmbSession.stub(:base_mount_point).and_return(Rails.root.join("/tmp"))
+      allow_any_instance_of(MiqSmbSession).to receive(:settings_mount_point).and_return(Rails.root.join("tmp"))
```

Comment:
Yes, I added them in `gems/pending/spec/util/mount/miq_generic_mount_session_spec.rb`